tool/gravity/cli/run.go

```diff
@@ -136,6 +141,11 @@ func InitAndCheck(g *Application, cmd string) error {
 				utils.InitLogging(defaults.GravitySystemLogFile)
 			}
 		}
+	default:
+		if systemLogSet {
+			// For all commands, use the system log file explcitly set on command line
```

Comment:
explicitly

Reply:
Also, will this still write to the default location? I think it should.

Reply:
Not sure I follow - do you mean write the log regardless `if systemLogSet`?